### PR TITLE
Added support to run test over GCP

### PR DESCRIPTION
**Pull request by anubhav-here**
Signed-off-by: Anubhav Deep <adeep@redhat.com>

### CODE REVIEW

File: conf/deployment/gcp/ipi_1az_rhcos_3m_3w.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+ENV_DATA:
+  platform: 'gcp'
+  deployment_type: 'ipi'
+  worker_replicas: 3
+  master_replicas: 3
+  master_instance_type: 'n1-standard-16'
+  worker_instance_type: 'n1-standard-16'
```
Comment (prsurve) [inline conf/deployment/gcp/ipi_1az_rhcos_3m_3w.yaml:8]: are we using same instance type for master and worker?

Comment (anubhav-here) [reply conf/deployment/gcp/ipi_1az_rhcos_3m_3w.yaml:8]: As of now, Yes!

**Comment by prsurve** [comment]
Please add GCP here https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L634

**Comment by ebenahar** [comment]
> Please add GCP here https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L634

+1!

**Comment by anubhav-here** [comment]
> > Please add GCP here https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L634
> 
> +1!

Added! @prsurve @ebenahar

**Pull request merged**